client/src/components/ItemSummary.jsx

```diff
@@ -4,7 +4,7 @@ import styled from 'styled-components';
 import NameAndPrice from './NameAndPrice.jsx';
 import Description from './Description.jsx';
 import ColorSelector from './ColorSelector.jsx';
-import Accordions from './Accordions.jsx';
+import Accordion from './Accordions.jsx';
```

Comment:
Accordion is singular while Accordions.jsx is plural--intentional?

Reply:
Good eye! I can't believe it hasn't bit me yet